test/functional/apps/visualize/_visualize_screenshots.js

```diff
@@ -0,0 +1,247 @@
+import expect from 'expect.js';
+
+export default function ({ getService, getPageObjects, updateBaselines }) {
+  const log = getService('log');
+  const esArchiver = getService('esArchiver');
+  const screenshot = getService('screenshots');
+  const PageObjects = getPageObjects(['common', 'visualize', 'header']);
+  const remote = getService('remote');
+
+
+  describe('visualize app', function describeIndexTests() {
```

Comment:
Change `visualize app` to something unique for this test suite (file) like `visualize screenshots`.
The parent index.js already has a describe with `visualize app`.
To make it easier to run just this test suite with --grep, we should have a unique describe in each test suite.